pkg/sql/conn_executor.go

```diff
@@ -1920,6 +1919,7 @@ func (ex *connExecutor) resetExtraTxnState(ctx context.Context, ev txnEvent) {
 	ex.extraTxnState.numDDL = 0
 	ex.extraTxnState.firstStmtExecuted = false
 	ex.extraTxnState.hasAdminRoleCache = HasAdminRoleCache{}
+	ex.extraTxnState.createdSequences = nil
```

Comment:
Is it safe to move this zeroing to an earlier place in the flow?

Reply:
It should be. No one is calling `isCreatedSequence` in the succeeding lines. I moved the line up to keep this next to `extraTxnState.hasAdminRoleCache` (its neighbor in the struct definition and in `newConnExecutor`), but I'm happy to move it back if you'd like.

Reply:
That's fine I just wanted to make sure it was not moved by mistake.